src/components/Event/RenderList/RenderList.jsx

```diff
@@ -1,59 +1,64 @@
-﻿import React from "react";
+import React from "react";
 import PropTypes from "prop-types";
-import renderListConstants from "../../../constants/renderListsConstants";
-import PagePagination from "../../shared/PagePagination/PagePagination";
-
-const { NO_RESULTS } = renderListConstants;
+import Pagination from "@material-ui/lab/Pagination";
+import SpinnerWrapper from "../../../containers/SpinnerContainer/SpinnerContainer";
+import NoResult from "../../shared/NoResult/NoResult";
+import "./RenderList.scss";
+import { EMPTY_DRAFT } from "../../../constants/draftConstants";
+import DraftEventCard from "../../Draft/DraftEventCard";
 
 const RenderList = ({
   page,
   totalPages,
   dataList,
-  customNoResultsMessage,
-  renderSingleItem,
   handlePageChange,
+  onDelete,
 }) => {
-  const renderItems = arr => arr.map(item => renderSingleItem(item));
+  const pageChange = (event, value) => {
+    handlePageChange(value);
+  };
 
   return (
-    <>
-      <div className="row">
-        {dataList.length ? (
-          renderItems(dataList)
-        ) : (
-          <div id="notfound" className="w-100 notfound mw-100 notfound-404 h1">
-            {customNoResultsMessage || NO_RESULTS}
+    <div className="container">
+      <SpinnerWrapper showContent={dataList !== null}>
+        {dataList.length > 0 ? (
```

Comment:
maybe create variables for this and above checks?

Reply:
fixed